unafamilia/src/main/java/com/unfamilia/application/order/OrderController.java

```diff
@@ -58,7 +58,7 @@ public Response newOrder(OrderPayload orderPayload, @PathParam("userId") Long us
                         player.getDiscordUserId(),
                         orderId,
                         orderPayload.getItems().stream()
-                                .map(item -> new NewOrderCommand.OrderedItem(item.itemName(), item.quantity()))
+                                .map(item -> new NewOrderCommand.OrderedItem(item.item_id(), item.quantity()))

```

Comment:
Nice! But did you remember to change the type and name of `OrderPayload.OrderedItem.name` parameter? This might be potentially dangerous and breeds mistakes